app/src/androidTest/java/comcmput301f16t01/github/carrier/SearchingTests.java

```diff
@@ -2,55 +2,34 @@
 
 import junit.framework.Assert;
 
-import java.util.ArrayList;
-
-
-import comcmput301f16t01.github.carrier.Notifications.ElasticNotificationController;
 import comcmput301f16t01.github.carrier.Requests.ElasticRequestController;
 import comcmput301f16t01.github.carrier.Requests.Request;
 import comcmput301f16t01.github.carrier.Requests.RequestController;
 import comcmput301f16t01.github.carrier.Requests.RequestList;
 
 
 public class SearchingTests extends ApplicationTest {
-    // University of Alberta, Edmonton
-    static final double latitude1 = 53.5232;
-    static final double longitude1 = -113.5263;
+    // somewhere in Tokyo, Japan
+    static final double latitude1 = 35.6895;
+    static final double longitude1 = 139.6917;
 
-    // somewhere in London, Ontario
-    static final double latitude2 = 42.9870;
-    static final double longitude2 = -81.2432;
+    // somewhere in Seoul, South Korea
+    static final double latitude2 = 37.5665;
+    static final double longitude2 = 126.9780;
 
-    // somewhere in St. Albert, Alberta
-    static final double latitude3 = 53.6305;
-    static final double longitude3 = -113.6256;
+    // somewhere in Kawasaki, Japan
+    static final double latitude3 = 35.5308;
+    static final double longitude3 = 139.7030;
 
-    // somewhere in Edmonton, Alberta
-    static final double latitude4 = 53.5444;
-    static final double longitude4 = -113.4909;
+    // Imperial Palace, Tokyo, Japan
+    static final double latitude4 = 35.6852;
+    static final double longitude4 = 139.7528;
 
     private User loggedInUser = new User( "notifTestUser", "notify@email.com", "888-999-1234" );
     private User driverOne = new User( "notifTestDriver", "notifyYou@email.com", "0118-99-112" );
 
-    // Set up a test user to receive notifications
-    private void setUpUser() {
-        UserController uc = new UserController();
-        String result = uc.createNewUser( loggedInUser.getUsername(),
-                loggedInUser.getEmail(),
-                loggedInUser.getPhone() );
-
-        if (result == null) {
-            System.out.print( "null line" );
-        }
-
-        assertTrue( "Failed to log in for test.", uc.logInUser( loggedInUser.getUsername() ) );
-    }
-
     // This tear down method may not be working entirely as expected...test further
     protected void tearDown() throws Exception {
```

Comment:
Is it working as expected?
Can you add a comment as to what your tearDown() method does?